### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by blackms** [rastapasta/pokemon-go-mitm#165]
Can we have a working example about a gym attack?

**Comment by rastapasta** [comment]
What would you like to see this example do? 😄

**Comment by blackms** [comment]
How to generate correctly Attack Actions, I'm having problems there :)

**Comment by l3uddz** [comment]
Yeah, it seems no matter what is sent attack action wise, the defender receives 0 dmg

**Comment by trisk** [comment]
You can't alter or create any requests right now, because of the validation.

**Comment by blackms** [comment]
I know, but I'm using Rocket API, so the signature is OK. I'm missing just the mechanics and I was trying to understand the logic using the MITM client.
As @l3uddz said, we are able to engage the battle, but not to make damages.

**Comment by rastapasta** [comment]
@l3uddz @blackms ah, got you - can't you just use the logTraffic and enable the ignore of unneeded requests? this should make the communication pretty simple to understand. Cheers!

**Comment by blackms** [comment]
@rastapasta No, I'm having problems in understand the logic behind the battleaction sequence and timing.
